crons/breaking_news.go

```diff
@@ -64,10 +65,11 @@ func (cron *BreakingNewsCron) Execute() string {
 		log.Infof("polled %d tweets, last tweet id: %d", len(tweets), cron.lastTweetID)
 	}
 
-	texts := []string{}
+	var texts []string
+	var linkRegex = regexp.MustCompile(`https://[\s\S]+$`)
```

Comment:
```suggestion
	texts := []string{}
	linkRegex := regexp.MustCompile(`https://[\s\S]+$`)
```
자잘한건데, 다른곳에서도 바로 값을 저장하는 경우에는`:=` 쓰고 값이 다른곳에서 정해질때 var로 쓰고있어서 통일하면 좋을것 같아요.

Reply:
지금 보니까 goland의 warning이 떠서 고쳤던 건데 이런 말이 있네요. go를 잘 아는건 아니여서 판단이 안서는데 괜찮은 걸까요?https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices

Reply:
아하 이런게 있었군요. 여기선 어떻게 해도 상관없을것 같은데, texts만 var []string으로 해도 좋을 것 같아요!